### PR TITLE
feat(iamx): fetchBatch workflow

### PR DESCRIPTION
**Pull request by faiz-alhadiid**

### CODE REVIEW

File: lib/core/Executor.js [hunk 1]
```diff
@@ -64,6 +65,16 @@ exports.Executor = class Executor {
     return this._execShow(context);
   };
 
+  /**
+   * List available member from target platform
+   *
+   * @param {Object} context
+   */
+  fetchBatch (context) {
+    this._validateConnectorSupport(FETCH_BATCH);
```
Comment (mikeadityas) [inline lib/core/Executor.js:74]: Should we validate the context here?

File: lib/core/Executor.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 
-const PROVISION = 'provision';
-const REVOKE    = 'revoke';
-const SHOW      = 'show';
-const VERSION   = '0.0.1';
+const PROVISION   = 'provision';
+const REVOKE      = 'revoke';
+const SHOW        = 'show';
+const FETCH_BATCH = 'fetchBatch';
+const VERSION     = '0.0.1';
```
Comment (mikeadityas) [inline lib/core/Executor.js:7]: I think we need to bump this version.
But, this version is different from the one defined in `package.json` https://github.com/cermati/iamx/blob/master/package.json#L3
You might want to confirm with @sdsdkkk regarding the versioning.

File: lib/core/Executor.js (hunk 1 above)
Comment (faiz-alhadiid) [reply lib/core/Executor.js:74]: fetchBatch context actually differ from ReadOnlyAccessContext. So, i don't think we can validate the context using that.

Comment (mikeadityas) [reply lib/core/Executor.js:74]: Yeah, we need a different context for validation. Please add one ya.

Comment (faiz-alhadiid) [reply lib/core/Executor.js:74]: added new method `listContextFormat()`

File: lib/core/Executor.js (hunk 2 above)
Comment (faiz-alhadiid) [reply lib/core/Executor.js]: updated to `2.0,.0` for this VERSION and package version

**Pull request merged**